docs/UserGuide.md

```diff
@@ -69,13 +69,14 @@ Adds the details of a student to Vax@NUS records. It is mandatory to include the
 
 In addition, it is optional to include the following detail of a student:
 
+
 * School Residence, to determine if the student lives on campus. This will help NUS campus residences determine the proportion of vaccinated student residents using the `stats` command. 
 
 :information_source: **NOTE** If the School Residence of a student is not specified, the system will default to DOES_NOT_LIVE_ON_CAMPUS and assume that the student does not live on campus.
 
 > For a smooth user experience, please refer to the  please refer to the [Input Formats](#input-formats) section below for more information regarding input formats. That section explains which prefix should be used for each piece of information and how the information should be presented to the program. 
 
-`Add` Command Format: `add n/NAME i/MATRICULATION_NUMBER f/FACULTY p/PHONE_NUMBER e/EMAIL a/ADDRESS s/VACCINATION_STATUS m/MEDICAL_DETAILS r/SCHOOL_RESIDENCE[optional]`
+`Add` Command Format: `add n/NAME i/MATRICULATION_NUMBER f/FACULTY p/PHONE_NUMBER e/EMAIL a/ADDRESS s/VACCINATION_STATUS m/MEDICAL_DETAILS [r/SCHOOL_RESIDENCE]`
```

Comment:
Sorry, just to clarify, is SCHOOL_RESIDENCE now optional?

Reply:
Yea, it is still an optional field.. Earlier on in our UG, we state that items  in square bracket are optional so we should stil follow their convection!

Reply:
Apologies for the oversight! I understand now.